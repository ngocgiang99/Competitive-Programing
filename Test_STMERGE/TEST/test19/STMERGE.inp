7
1 3
775019863 887792013 530991382 
4 7
995174872 358760490 918931259 340498595 811049912 707023964 281102874 
764524914 587561840 50393476 686254477 208684087 58750343 930612660 
999434809 65910056 350587585 416448323 664404738 149764794 819634723 
882348229 534131470 490141090 502826738 659033360 33964994 731493077 
1 6
567333058 78970247 665961772 564028563 725793780 936581423 
2 10
296533372 613463936 647991958 690862963 660069312 346874993 228535890 108402402 557717861 415442485 
813509209 533761880 879268687 309938498 680043952 586120042 214135461 264835149 593841564 594763244 
5 3
364513186 481204211 918488419 
957016689 739926408 785720532 
307008086 929063475 692752210 
448054821 592999367 509525320 
296136517 351646651 360124444 
5 7
134248874 328931834 481873275 286175226 732819527 732439110 978245443 
533715078 404023335 917968420 937305009 525920463 230637088 450722042 
829632954 537913435 409825095 853344876 953273491 164226797 228382447 
955531360 190322609 881856652 502549167 217821435 37716680 115123580 
416089692 831119010 741251023 786195522 125358090 658875385 556181753 
5 10
580462053 343029437 359023269 933855486 254651789 664281038 832102539 707579310 408454228 804761890 
423222492 844329267 416601567 63883758 851522396 286172387 908594196 586702759 942617104 690277459 
355585304 744905218 445231421 706398038 133807952 809834228 44798312 415744515 433688928 159753905 
395747158 254524591 544301614 295045552 359443469 783402989 401239774 655633060 215248 412249237 
433924310 944944259 859357990 978318777 721723835 936521859 675262163 377363120 23574385 839652887 
