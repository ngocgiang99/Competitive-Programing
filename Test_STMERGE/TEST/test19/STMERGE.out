530991382
281102874
567333058
415442485
296136517
411533316
433924310
